rtbkit/core/router/router.cc

```diff
@@ -1417,8 +1451,8 @@ augmentAuction(const std::shared_ptr<AugmentationInfo> & info)
             wakeupMainLoop.signal();
         };
 
-    augmentationLoop.augment(info, Date::now().plusSeconds(augmentationWindow.count()),
-                             onDoneAugmenting);
+    augmentationLoops.at(numAuctions % augmentationLoops.size())->augment(
```

Comment:
I would avoid using `at()` in this case, using `operator[]` is safe enough as you're clearly controlling the bound with the modulo so you don't need any additional bound checking.